File: Day-19/ProductAndSum.py
import math
n = int(input())
def subtractProductAndSum(n):
    num= n
    product = 1
    sum = 0
    while(num):
        product *= num % 10
        sum += num % 10
        num = math.floor(num / 10)
            
    return product - sum
        
print(subtractProductAndSum(n))

File: Day-19/ProductAndSum.js
function ProductAndSum(n) {
  let num = n;
  let product = 1;
  let sum = 0;

  while (num) {
    let unit = num % 10;
    product *= unit;
    sum += unit;
    num = Math.floor(num / 10);
  }
  return product - sum;
}

console.log(ProductAndSum(234));
